hbase-client/src/main/java/org/apache/hadoop/hbase/client/Table.java

```diff
@@ -791,7 +794,11 @@ default int getRpcTimeout() {
    *
    * @param rpcTimeout the timeout of each rpc request in millisecond.
    *
-   * @deprecated Use setReadRpcTimeout or setWriteRpcTimeout instead
+   * @deprecated since 1.4.0 and will be removed in 3.0.0. Use {@link #setReadRpcTimeout(int)} or
```

Comment:
This seems a little strange. Was there a 1.4.0 when 2.0.0 was released?

Reply:
Looks like, as far as I can see in the Git history. The commit to 2.0.0 was 30d7eeaefe431bc263519064662e6dc8ad8ff05a and to 1.4.0 the commit was ec99838b9c5ac931b16e7cc3c7c2b7bf5b024c0c.

Reply:
Great.